src/components/views/elements/TooltipTarget.tsx

```diff
@@ -60,10 +61,7 @@ const TooltipTarget: React.FC<IProps> = ({
             tabIndex={0}
             aria-describedby={id}
             className={tooltipTargetClassName}
-            onMouseOver={show}
-            onMouseLeave={hide}
-            onFocus={show}
-            onBlur={hide}
```

Comment:
Not sure how big a fan I am of using native DOM event listeners instead of using the tools React gave us for this purpose

Reply:
@matrix-org/element-web do we have any opinions here?

Reply:
(it might actually help fixing the tests)

Reply:
Not to my knowledge

Reply:
I guess the hook could return a `props` object that would get destructured then though I think it won't get much nicer-looking than that